tket/include/tket/Transformations/GreedyPauliOptimisation.hpp

```diff
@@ -525,8 +525,11 @@ typedef boost::adj_list_vertex_property_map<
  */
 class GPGraph {
  public:
-  /** Construct an GPGraph from a circuit */
-  GPGraph(const Circuit& circ);
+  /** Construct an GPGraph from the circuit size */
+  GPGraph(qubit_vector_t qubits, bit_vector_t bits);
+
+  GPGraph get_graph(
```

Comment:
Can you document what this function does?

Reply:
Ah, that method shouldn't even be there & is a leftover from an alternative way I considered doing this. I've removed it!